source/libs/page-detect.js

```diff
@@ -86,7 +86,11 @@ export const isPRSearch = () => location.pathname.startsWith('/pulls');
 
 export const isQuickPR = () => isCompare() && /[?&]quick_pull=1(&|$)/.test(location.search);
 
-export const isReleases = () => /^(releases|tags)/.test(getRepoPath());
+export const isReleases = () => /^releases/.test(getRepoPath());
+
+export const isTags = () => /^tags/.test(getRepoPath());
+
+export const isReleasesOrTags = () => isReleases() || isTags();
```

Comment:
Why? We don't use isReleases or isTags separately anywhere

Reply:
Someone might need to use it and create a bug as the name is misleading.

Reply:
> Someone might need to use it

They'll add it if they'll need it. Currently two of these new `is` aren't tested :D 

It can just be renamed (or you can add the tests)